log4j-core/src/main/java/org/apache/logging/log4j/core/layout/HtmlLayout.java

```diff
@@ -345,14 +355,17 @@ public static HtmlLayout createLayout(
             @PluginAttribute String contentType,
             @PluginAttribute(defaultString = "UTF-8") final Charset charset,
             @PluginAttribute String fontSize,
-            @PluginAttribute(value = "fontName", defaultString = DEFAULT_FONT_FAMILY) final String font) {
+            @PluginAttribute(value = "fontName", defaultString = DEFAULT_FONT_FAMILY) final String font,
```

Comment:
This factory method isn't needed anymore since the builder class is present.

Reply:
OK, this method has been deleted. Please review again.